### DISCUSSION TITLE
Mods Folder

### DISCUSSION THREAD

**Discussion by NotStampyC** [FunkinCrew/Funkin#2475] [Q&A] [answered]
When i checked out the mods folder, it no longer has introMod inside it in both the prototype weekend version and main weekend.
DOes anyone know if it actualy does something now. 
Like is it similar to psych engine or am i being an idiot

**Comment by moxie-coder** [comment]
yeah it does something now

**Reply by NotStampyC** [reply]
yea but what tho

**Reply by biomseed** [reply]
It's a fully functional polymod system. Look up polymod and you can see the documentation of it.

**Reply by NotStampyC** [reply]
how does it work tho
i cant make sense of it

**Reply by biomseed** [reply, marked as answer]
read the polymod documentation, it tells you how it works and the game already has everything set up for you, you can look at the scripts that are already in the assets folder of the game as reference because a lot of things have moved to hscript, and documentation for the json files are in the "data" structures in the source code
